### PR TITLE
Standard

### PR DESCRIPTION
**Pull request by pguth**

### CODE REVIEW

File: bin.js [hunk 1]
```diff
@@ -44,23 +44,31 @@ if (cmd === 'listen') {
   server.listen(argv.port, argv.host, function () {
     console.log('signalhub listening on port %d', server.address().port)
   })
-  return
+  process.exit(0)
```
Comment (mafintosh) [inline bin.js]: this unfortunately won't work since that kills the process and therefor the server :(

File: bin.js [hunk 2]
```diff
@@ -44,23 +44,31 @@ if (cmd === 'listen') {
...
+  if (argv._.length < 3) {
+    console.error('Usage: signalhub subscribe [app] [channel]')
+    process.exit(1)
+  }
+  client = require('./')(argv._[1], argv.host + ':' + argv.port || 'localhost:8080')
   client.subscribe(argv._[2]).on('data', function (data) {
     console.log(data)
   })
-  return
+  process.exit(0)
```
Comment (mafintosh) [inline bin.js]: same as above

File: bin.js [hunk 3]
```diff
@@ -44,23 +44,31 @@ if (cmd === 'listen') {
...
-  if (argv._.length < 4) return console.error('Usage: signalhub broadcast [app] [channel] [json-message]')
-  var client = require('./')(argv._[1], argv.host + ':' + argv.port || 'localhost:8080')
+  if (argv._.length < 4) {
+    console.error('Usage: signalhub broadcast [app] [channel] [json-message]')
+    process.exit(1)
+  }
+  client = require('./')(argv._[1], argv.host + ':' + argv.port || 'localhost:8080')
   client.broadcast(argv._[2], JSON.parse(argv._[3]))
-  return
+  process.exit(0)
```
Comment (mafintosh) [inline bin.js]: same as above

**Comment by maxogden** [comment]
maybe just wrap that code in a function so standard doesnt complain about it?

**Comment by feross** [comment]
Not supporting `return` at the top level is an unfortunate eslint artifact. Sorry guys :(

**Comment by pguth** [comment]
Yes, looks better @maxogden!

**Comment by pguth** [comment]
Whops. Closed by accident.

**Pull request closed without merging**